interface/main/tabs/templates/patient_data_template.php

```diff
@@ -37,7 +37,7 @@ class="img-thumbnail"
                     </a>
                 <!-- /ko -->
                 <!-- ko ifnot: patient -->
-                    <?php echo xlt("None{{Patient}}");?>
+                    <?php echo xlt("None"). xlt("{{Patient}}");?>
```

Comment:
These are part of the translation constant. They are comments letting translating know a context if needed. It will translate if there is a None{{Patient}} that has been translated. That reminds me; it's getting near time to do another translation run to get all these new constants into the translation spreadsheet.